### PR TITLE
Auctions2 QA Fixes

### PR DESCRIPTION
**Pull request by The-Beez-Kneez**
This addresses the following tickets:
- https://artsyproduct.atlassian.net/browse/GRO-267
- https://artsyproduct.atlassian.net/browse/GRO-264
- https://artsyproduct.atlassian.net/browse/GRO-266

This is for Auctions2 QA fixes. I have updated the "Learn more about bidding on Artsy" to direct to Zendesk help center.  Updated the introduction copy text on artsy.net/auctions2. And put in a filter sort for Current Auctions so user sees auctions in chronological order.

Screen Shot 2021-04-05 at 10 40 26 AM

Screen Shot 2021-04-05 at 8 06 37 AM

### CODE REVIEW

**Review by damassi — APPROVED** [review]
👍 👍

File: src/v2/Apps/Auctions/AuctionsApp.tsx [hunk 1]
```diff
@@ -29,14 +29,17 @@ const AuctionsApp: React.FC<AuctionsAppProps> = props => {
...
+          Bid on works you love with auctions on Artsy. With bidding opening
+          daily, Artsy connects collectors like you to art from leading auction
+          houses, nonprofit organizations, and sellers across the globe. We
+          feature premium artworks including modern, contemporary, and street
+          art, so you can find works by your favorite artists—and discover new
+          ones—all in one place.
         </Text>
-        <RouterLink to="/how-auctions-work" noUnderline>
+        <RouterLink
+          to="https://support.artsy.net/hc/en-us/sections/360008298773-Bid-at-Auction"
```
Comment (damassi) [inline src/v2/Apps/Auctions/AuctionsApp.tsx:40]: Can we add a test for this url [this the test file?](https://github.com/artsy/force/blob/be080de2f12ba00bcaa33ea8fa4165729687125c/src/v2/Apps/Auctions/__tests__/AuctionsApp.jest.tsx)

**Review by zephraph — APPROVED** [review]

**Comment by zephraph** [comment]
Hey @The-Beez-Kneez, this looks good to me. I re-assigned it to you (in case you want to address @damassi feedback above). Feel free to merge when you're ready.

File: src/v2/Apps/Auctions/AuctionsApp.tsx (hunk 1 above)
Comment (The-Beez-Kneez) [reply src/v2/Apps/Auctions/AuctionsApp.tsx:40]: Added in a jest test for this!

**Review by damassi — APPROVED** [review]

**Pull request merged**